frontend/ui-pablo/app/defi/config/index.ts

```diff
@@ -0,0 +1,56 @@
+export const DEFI_CONFIG = {
+    networkIds: [1, 137, 42161, 43114, 1285, 250] as const, // important
+    tokenIds: [
+      "eth",
+      "matic",
+      "avax",
+      "weth",
+      "usdc",
+      "dot",
+      "uni",
+      "ftm",
+      "pica",
+      "movr",
+      "ksm",
+      "pablo",
+      "chaos",
+    ] as const, // important
+    ammIds: ["uniswap", "curve", "balancer"] as const,
+    swapChartIntervals: [
+      {
+        symbol: "24h",
+        name: "24 hours",
+      },
+      {
+        symbol: "1w",
+        name: "1 week",
+      },
+      {
+        symbol: "1m",
+        name: "1 month",
+      },
+      // {
+      //   symbol: "1y",
+      //   name: "1 year",
+      // },
```

Comment:
Is it useful to have this comment?

Reply:
In case we want to enable 1y charts we can just uncomment this line.